src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/controller/UnallocatedCasesController.kt

```diff
@@ -6,28 +6,43 @@ import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsallocations.jpa.entity.repository.UnallocatedCaseEntity
+import uk.gov.justice.digital.hmpps.hmppsallocations.jpa.repository.UnallocatedCasesRepository
+import java.time.LocalDateTime
 
 @RestController
 @RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
I have few little issue with the file. The first one is that the Repository layer should be incapsulated in a Service one. A data layer package should be created and so a service layer one. We should favor constructor injection over field one and force the controller to inject the service layer.